src/battle_script_commands.c

```diff
@@ -11384,7 +11384,13 @@ static u32 ChangeStatBuffs(s8 statValue, u32 statId, u32 flags, const u8 *BS_ptr
         else if ((battlerHoldEffect == HOLD_EFFECT_CLEAR_AMULET
                   || battlerAbility == ABILITY_CLEAR_BODY
                   || battlerAbility == ABILITY_FULL_METAL_BODY
-                  || battlerAbility == ABILITY_WHITE_SMOKE)
+                  || battlerAbility == ABILITY_WHITE_SMOKE
+                  || (battlerAbility == ABILITY_HYPER_CUTTER && statId == STAT_ATK))
+                  || ((B_UPDATED_INTIMIDATE >= GEN_8) && statId == STAT_ATK
+                 && (battlerAbility == ABILITY_INNER_FOCUS
+                  || battlerAbility == ABILITY_SCRAPPY
+                  || battlerAbility == ABILITY_OWN_TEMPO
+                  || battlerAbility == ABILITY_OBLIVIOUS))
                  && (!affectsUser || mirrorArmored) && !certain && gCurrentMove != MOVE_CURSE)
```

Comment:
The parentheses here are wrong.

![notepad++_20231019_165457068](https://github.com/rh-hideout/pokeemerald-expansion/assets/4485172/992db85c-a08c-42d9-963c-105d5bba4619)

Honestly though, this statement is quite a handful. It may be a good idea to add an array of abilities like the ones used for Mold Breaker, Trace, etc to make this more readable and palatable :eyes:

Reply:
Yeah an array might be good but I'm starting to realize that what I did is wrong. The second half is supposed to be triggered on Intimidate, not if the attack stat is lowered except Hyper Cutter but I removed that one in a later commit since it was already checked.

Reply:
Ended up making a function with a switch because the array looked over engineered for what it needed to do.